bin/eslint-formatters-calypso/src/lib/parsing-errors.js

```diff
@@ -0,0 +1,36 @@
+/* eslint strict: "off" */
+
+'use strict';
+
+module.exports = function( report ) {
+	// deep clone the report, so we can create a new one
+	// to tweak and edit in place - as this function remains pure.
+	let newReport = [];
+
+	const isParsingError = ( message ) => {
+		if ( ( message.ruleId === null ) &&
+		( message.severity === 2 ) &&
+		( message.message.startsWith( 'Parsing error:' ) ) ) {
```

Comment:
I have been thought more about this. I have added some comments with the current consensus within ESLint team regarding `parsing-errors` (see [discussion](https://github.com/eslint/eslint/issues/3555) and [PR](https://github.com/eslint/eslint/pull/3967)) and have modified this to check for `ruleId=null` **or** `message.startsWith=Parsing error` **or** `fatal` key.

Although the later approach is correct for the current state of ESLint, this way we are more liberal in reporting parsing-errors and would be more safe if the ESLint API changes and those changes go undetected when Calypso updates its ESLint package. I cannot see a future-proof approach, so I think this is the better we can do.